### PR TITLE
Refactor UniquePersonList to UniqueApplicantList

### PR DESCRIPTION
**Pull request by SethCKL**
Rename UniquePersonList to UniqueApplicantList to reflect the change from Person class to Applicant class.

Hence the container of Persons previously should also be renamed

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
  */
 public class AddressBook implements ReadOnlyAddressBook {
 
-    private final UniquePersonList persons;
+    private final UniqueApplicantList persons;
```
Comment (likeabowx) [inline src/main/java/seedu/address/model/AddressBook.java:21]: Time to change the variable name too for consistency

File: src/main/java/seedu/address/model/applicant/UniqueApplicantList.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public void remove(Applicant toRemove) {
         }
     }
 
-    public void setPersons(UniquePersonList replacement) {
+    public void setPersons(UniqueApplicantList replacement) {
```
Comment (likeabowx) [inline src/main/java/seedu/address/model/applicant/UniqueApplicantList.java:82]: Can change the method names as well

**Review by likeabowx — CHANGES_REQUESTED** [review]
There's still a number of instances where the term `person` is used, including in variable names, method names, and comments. Just try to change as much as possible :)

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (SethCKL) [reply src/main/java/seedu/address/model/AddressBook.java]: Agreed. Thanks for pointing it out will work on resolving this

**Comment by SethCKL** [comment]
After thorough review, there are still other instances of Persons/Person found in the code base in other packages, classes and tests.

It's very pervasive and spans many classes as AB-3 is practically built on interacting and manipulating persons objects.

I will continue to refactor other instances of Persons/person over the coming days

**Review by likeabowx — APPROVED** [review]
LGTM, will need to slowly change the term `person` as it is really impossible to change all at once

**Pull request merged**